integration/Substra/assets/dataset/opener_test_MyOrg1MSP.py

```diff
@@ -0,0 +1,24 @@
+from flamby.datasets.fed_tcga_brca import FedTcgaBrca
+import substratools as tools
+from torch.utils.data import DataLoader
+
+
+class FlambyTestOpener(tools.Opener):
+    def get_X(self, folders):
+        config = self.get_config()
+        return config
+
+    def get_y(self, folders):
+        config = self.get_config()
+        dataset = FedTcgaBrca(**config)
+        dataloader = DataLoader(dataset, batch_size=len(dataset))
+        return next(iter(dataloader))[1]
+
+    def get_config(self):
+        return {"center": 0, "train": False}
+
+    def fake_X(self, n_samples=None):
```

Comment:
This should be implemented or deprecated in substra. This example should be adaptable to deployed network as well.